lib/checks/media/frame-tested.json

```diff
@@ -8,7 +8,11 @@
 		"impact": "critical",
 		"messages": {
 			"pass": "The iframe was tested with axe-core",
-			"fail": "The iframe could not be tested with axe-core",
+			"fail": {
+				"default": "The iframe could not be tested with axe-core",
+				"version": "The iframe is using a different version of axe;\nprimary version: ${data.version}\niframe version: ${data.iframeVersion}",
+				"configure": "The iframe is using a different configuration of axe;\nprimary config: ${data.config}\niframe config: ${data.iframeConfig}"
```

Comment:
I think axe-core should throw, instead of return a failed result if it gets a response from an iframe that has the wrong config or the wrong axe-core version. The issue is that by returning this as a failed result, we're still allowing invalid results to make it into our output. It also becomes harder for people integrating axe-core to notice this issue. They would have to add an assert to check that this violation was in the result, whereas if it threw an error, the problem would be very apparent. It seems like throwing is our best bet for preventing this from making it into any integration.